azure-sdk/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id 'io.micronaut.build.internal.azure-module'
+}
+
+dependencies {
+    annotationProcessor "io.micronaut:micronaut-inject-java"
+    implementation "io.micronaut:micronaut-inject-java"
+    implementation("io.micronaut:micronaut-runtime")
+    // todo ask cedric about best way how to enforce boms
+    implementation platform('com.azure:azure-sdk-bom:1.0.3')
```

Comment:
@melix what's the right way how to force external bom? aka how to add it to our bom

Reply:
Since we provide `micronaut-azure-bom`, we need to import the azure SDK bom into it. So edit the `azure-bom/build.gradle` file and do:

```
dependencies {
    constraints {
        api "com.microsoft.azure.functions:azure-functions-java-library:$azureFunctionsJavaLibraryVersion"
    }
    api platform("com.azure:azure-sdk-bom:1.0.3")
}
```

then you can have an implementation dependency on the BOM in this project like this (not tested):

```
implementation platform(project(":azure-bom"))
```